src/Command/DumpEnvCommand.php

```diff
@@ -127,7 +127,7 @@ private function loadEnv(string $path, string $env): array
         $process->run();
 
         if (!$process->isSuccessful()) {
-            throw new \RuntimeException($process->getErrorOutput());
+            throw new \RuntimeException($process->getErrorOutput().$process->getOutput());
```

Comment:
1. there is a ProcessException that does that, but in a better way
2. but there is the `mustRun()` method that is even simpler

https://github.com/symfony/flex/pull/496